src/transaction.js

```diff
@@ -99,7 +99,7 @@ export class Transaction extends TransactionBase {
         source = StrKey.encodeEd25519PublicKey(this.tx.sourceAccountEd25519());
         break;
       default:
-        source = this._muxedToString(this.tx.sourceAccount());
+        source = StrKey.encodeMuxedAccount(this.tx.sourceAccount().toXDR());
```

Comment:
should the source string be calculated in the constructor like some of the other properties in this class?

Reply:
@tamirms  I moved it here to experiment with a different pattern, but actually let's keep it in the source. It is consistent with the other places.